package org.folio.cataloging.exception;

public class RecordInUseException extends LibrisuiteException {

}
